core/lib/Thelia/ImportExport/Export/ExportHandler.php

```diff
@@ -198,6 +201,16 @@ public function isImageExport()
     }
 
     /**
+     * Get filename
+     *
+     * @return array
+     */
+    public function getFilename()
+    {
+        return $this::DEFAULT_FILENAME;
```

Comment:
I'd rather use the ```static``` keyword instead of ```$this```. It's more explicit that we are accessing a constant dynamically